guides-for-using-cinchy/builder-guides/cinchydxd-utility/overview.md

```diff
@@ -0,0 +1,61 @@
+# CinchyDXD
+
+## CinchyDXD introduction
+
+CinchyDXD is a utility that exports a data experiences from a source Cinchy environment to a release package that you can:
+
+- Deploy to an upper environment with the same CinchyDXD utility.
+- Commit to source control to reflect a snapshot of a specific point in the release version.
+
+## About CinchyDXD 2.0
+
+CinchyDXD 2.0 creates packages for further development in a version controlled environment (such as Git) where multiple developers can work on the same data experience.
+
+## Prerequisites
```

Comment:
Just Thinking out loud. Does it make sense to embed the workflow overview here instead and then provide a link to each step?

1. Install DXD (link to the install instructions)
2. Package a data experience
  1. for the first time
  2. subsequent updates 
3. Install a data experience into another env

Reply:
I think this can exist at the bottom of the page. To me, the landing page should get you up and running and take you to other sections on how to work it. Good catch that there wasn't an overall journey placed on the landing page. I'll refactor it now.